package/yast2-storage.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Mon Apr  1 13:20:46 UTC 2019 - mvidner@suse.com
+
+- Fix "Not enough space available to propose separate /home."
+  (bsc#1113714)
+- 3.2.20.1
```

Comment:
Why adding a new digit? This is not diverging from any more modern branch, isn't it?

Reply:
Replied in IRC: is because an already existing divergence between this branch and `master_old` :-(

Reply:
Because even 3.2.20 was colliding with #295 ... which was the last existing maser version, so 3.2.21 would actually not collide. But version numbers are cheap.